_i18n/it/get-started/accepting.md

```diff
@@ -33,23 +32,23 @@ If you want to check for a payment using monero-wallet-cli you can use the "paym
 [wallet 49VNLa]: █
 ```
 
-If you need to check for payments programmatically, then details follow the next section.
+Se vuoi controllare i pagamenti in modo programmatico, nella prossima sezione trovi tutti i dettagli in merito.
 
-### Receiving a Payment Step-by-Step
+### Ricevere un pagamento passo dopo passo
 
-* Generate a random 64 character hexadecimal string for the payment  
-* Communicate the payment ID and Monero address to the individual who is making payment  
-* Check for the payment using the "payments" command in monero-wallet-cli
+* Genera una stringa casuale di 64 caratteri esadecimali per il pagamento (ID pagamento)
+* Comunica l'ID pagamento generato nonché il tuo indirizzo Monero a chi vuole effettuare un pagamento
+* A seguito del pagamento, controlla il suo stato usando il comando "payments" in monero-wallet-cli 
 
-### Checking for a Payment Programmatically
+### Controllare lo stato di un pagamento in modo programmatico
 
-In order to check for a payment programmatically you can use the get_payments or get_bulk_payments JSON RPC API calls.
+Per controllare lo stato dei pagamenti in modo programmatico puoi utilizzare le chiamate API JSON RPC. 
```

Comment:
please add the API referenced: `puoi utilizzare le chiamate API JSON RPC get_payments o get_bulk_payments`

Reply:
Oh, right. It was an oversight.